src/tests/ftest/daos_test/suite.yaml

```diff
@@ -59,6 +59,7 @@ server_config:
         - DD_MASK=mgmt,io,md,epc,rebuild,any
         - D_LOG_FILE_APPEND_PID=1
         - D_LOG_FLUSH=DEBUG
+        - FI_LOG_LEVEL=warn
```

Comment:
Don't you want to also set D_LOG_STDERR_IN_LOG=1 in order to merge libfabric debug logs in engine's log ??

Reply:
Yup, this should be included too, i was not aware of this, thanks!